### PR TITLE
common Preview annotation

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**
Library `org.jetbrains.compose.components:components-ui-tooling-preview`
**This library is subject to change in the future.**

Added library `org.jetbrains.compose.components:components-ui-tooling-preview:VERSION`, where VERSION - shoud will be Compose version.
Simple Preview without arguments and PreviewParameterProvider for future usage.

### CODE REVIEW

File: components/ui-tooling-preview/library/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
+    }
+}
+
+configureMavenPublication(
+    groupId = "org.jetbrains.compose.components",
+    artifactId = "ui-tooling-preview",
+    name = "Compose tooling library API. This library provides the API required to declare " +
```
Comment (igordmn) [inline components/ui-tooling-preview/library/build.gradle.kts:53]:
```suggestion
    name = "Experimental Compose Multiplatform tooling library API. This library provides the API required to declare " +
```

File: components/ui-tooling-preview/library/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
+    }
+}
+
+configureMavenPublication(
+    groupId = "org.jetbrains.compose.components",
```
Comment (igordmn) [inline components/ui-tooling-preview/library/build.gradle.kts:51]: We, users, and IDE devs should be aware that this is subject to change in the future. Please, mention that in the PR message (with mentioning that it is an experiemental library).

File: components/ui-tooling-preview/library/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    artifactId = "ui-tooling-preview",
```
Comment (igordmn) [inline components/ui-tooling-preview/library/build.gradle.kts:52]:
```suggestion
    artifactId = "components-ui-tooling-preview",
```
- we add "components-" prefix:
image

File: components/ui-tooling-preview/library/build.gradle.kts (hunk 2 above)
Comment (dima-avdeev-jb) [reply components/ui-tooling-preview/library/build.gradle.kts:51]: Done, mentioned

**Review by igordmn — APPROVED** [review]
LGTM, let's wait for the second approval

**Review by pjBooms — APPROVED** [review]

**Pull request merged**